CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing guidelines
+
+## How to start?
+
+Great, so you want to join the development!
+
+First, [set up a development environment](INSTALL.md).
+Since you're going to write new code, use the `docker-compose.dev.yml` method.
+
+If everything went right, the system should be accessible at `http://localhost:80`.
+
+## Development workflow
+
+We use a standard [github fork workflow](
+https://gist.github.com/Chaser324/ce0505fbed06b947d962).
+
+1. Fork the repository.
+
+2. Create a new branch. The name does not matter, but the recommended format
+  is `feature/xxx` or `fix/yyy`.
+
+3. Work on your changes!
+
+4. If possible, add a test or two to the `tests/` directory.
+
+5. TODO: autoformatter configuration
+
+6. When you feel like you're done, commit the files:
+
+```bash
+$ git add -A
+$ git status  # check if included files match your expectations
```

Comment:
I'd rather recommend adding all the files manually because it let's you actually spot if you have some random files from the runtime lying around which are not in `.gitignore` but also shouldn't be commited.

This is very subjective, tho. Depends on sb's workflow.